doc/sphinx/dg.rst

```diff
@@ -1,3 +1,4 @@
+=================
```

Comment:
not needed, see http://docutils.sourceforge.net/docs/user/rst/quickref.html#section-structure